src/core/gps/qgsnmeaconnection.cpp

```diff
@@ -363,8 +371,6 @@ void QgsNmeaConnection::processGsvSentence( const char *data, int len )
     // clear() on GGA
 
     // for determining when to graph sat info
-    mLastGPSInformation.satInfoComplete = ( result.pack_index == result.pack_count );
```

Comment:
What about for devices which don't send a ZDA sentence? (I just checked, and I don't receive these)

Reply:
Looking at this, ZDA is a time and date message, so I don't think using it to detect when the satellite info is complete is valid at all.

Perhaps the logic should be that as soon as a "GSV" sentence is received we consider the satellite info incomplete, until we receive the next non-GSV sentence?

Reply:
> What about for devices which don't send a ZDA sentence? (I just checked, and I don't receive these)

Generally it is from the configuration of the GNSS receiver that NMEA messages are enabled!

Reply:
> 

I had chosen ZDA for the simple reason that it is always the last in NMEA cycle!
Your GSV strategy is the simplest to implement.
There will be a 1 cycle delay in NMEA sequence.